modules/steam.nix

```diff
@@ -247,6 +247,39 @@ in
           '';
         };
 
+        autoStart = mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Whether to automatically launch the Steam Deck UI on boot.
+
+            No Display Managers may be enabled in conjunction with this option.
+          '';
+        };
+
+        user = mkOption {
+          type = types.nullOr types.str;
+          default = null;
+          description = ''
+            The user to run Steam with.
+
+            This user must be in the `networkmanager` group to be able to
+            complete the OOBE (Out-of-Box Experience).
```

Comment:
This should _not_ be true now that we're doing PAM correctly (and also I added a polkit rule to force it but we should back that out)

Reply:
Since Steam does `org.freedesktop.NetworkManager.settings.modify.system` instead of `org.freedesktop.NetworkManager.settings.modify.own` we still need either group membership or the rule. I modified the rule to only allow active sessions to bypass the authentication, and removed the power action rules since they aren't needed anymore.